camel/storages/graph_storages/neo4j_graph.py

```diff
@@ -583,3 +583,6 @@ def add_graph_elements(
                     ]
                 },
             )
+
+    def clear(self) -> None:
+        pass
```

Comment:
we can also add code to realise this method if it's required by the memory module